File: JS-Basics/For Loop/Exercise/06.js
function calculateActorPoints(input) {
    const actorName = input[0]; 
    let points = parseFloat(input[1]); 
    const numJudges = parseInt(input[2]); 
    const judgesData = input.slice(3); 

    for (let i = 0; i < numJudges; i++) {
        const judgeName = judgesData[i * 2]; 
        const judgePoints = parseFloat(judgesData[i * 2 + 1]); 

        points += (judgeName.length * judgePoints) / 2;

        if (points > 1250.5) {
            console.log(`Congratulations, ${actorName} got a nominee for leading role with ${points.toFixed(1)}!`);
            return;
        }
    }

    const neededPoints = (1250.5 - points).toFixed(1);
    console.log(`Sorry, ${actorName} you need ${neededPoints} more!`);
}

File: Basics/For Loop/Exercise/06.py
actor = input()
points_from_academy = float(input())
count_jury = int(input())

jury_points = 0
all_points = 0
for i in range(1, count_jury + 1):
    name_jury = input()
    points_from_jury = float(input())
    jury_points += (len(name_jury) * points_from_jury) / 2
    all_points = points_from_academy + jury_points
    if all_points >= 1250.5:
        break
if all_points >= 1250.5:
    print(f"Congratulations, {actor} got a nominee for leading role with {all_points:.1f}!")
else:
    print(f"Sorry, {actor} you need {(1250.5 - all_points):.1f} more!")
